src/NuGetPackageVerifier/Rules/AssemblyStrongNameRule.cs

```diff
@@ -33,7 +37,7 @@ public IEnumerable<PackageVerifierIssue> Validate(PackageAnalysisContext context
                     try
                     {
                         using (var packageFileStream = context.PackageReader.GetStream(currentFile))
-                        using (var fileStream = File.OpenWrite(assemblyPath))
+                        using (var fileStream = new FileStream(assemblyPath, FileMode.Create))
```

Comment:
Why was this change necessary? `OpenWrite` creates the file if it doesn't exist.